fenix/app/src/main/java/org/mozilla/fenix/messaging/FenixNimbusMessagingController.kt

```diff
@@ -0,0 +1,19 @@
+package org.mozilla.fenix.messaging
+
+import mozilla.components.service.nimbus.messaging.NimbusMessagingController
+import mozilla.components.service.nimbus.messaging.NimbusMessagingStorage
+import org.mozilla.fenix.BuildConfig
+
+/**
+ * Bookkeeping for message actions in terms of Glean messages and the messaging store.
+ * Specialized implementation of NimbusMessagingController defining the deepLinkScheme
+ * used by Fenix
+ */
+class FenixNimbusMessagingController(
+    messagingStorage: NimbusMessagingStorage,
+    now: () -> Long = { System.currentTimeMillis() },
+) : NimbusMessagingController(
+    messagingStorage = messagingStorage,
+    deepLinkScheme = BuildConfig.DEEP_LINK_SCHEME,
+    now = now,
+)
```

Comment:
Maybe, we may not need a new specific type of `NimbusMessagingController`? As we are not adding new behaviour just passing values to the super class. Maybe there is an use case that I'm missing could you help me to clarify :)

Reply:
The only reason I created this specific type was to unsure that Fenix uses NimbusMessagingController with the same configuration everywhere, from the app code and from the test code.  
For example, I wanted to avoid the further possibility of specifying a specific `httpActionToDeepLinkUriConvertor` somewhere and forgetting to do it all across the app. Does it make sense? 

Reply:
Got it!
We could also create a single instance on `Analytics`, and reduce it everywhere else, similar to what we do with [messagingStorage](https://github.com/mozilla-mobile/firefox-android/blob/5bfbab24d37489c001da8f63471c3e22f8400a63/fenix/app/src/main/java/org/mozilla/fenix/components/Analytics.kt#L149-L160).